src/templates/next/components/KeyStatistics/KeyStatistics.tsx

```diff
@@ -19,7 +19,9 @@ const KeyStatistics = ({ variant, title, statistics }: KeyStatisticsProps) => {
           .slice(0, variant === "side" ? 3 : 4)
           .map(({ label, value }) => (
             <div className="flex flex-col gap-3">
-              <h3 className="text-5xl leading-[3.5rem]">{value}</h3>
+              <h3 className="text-5xl leading-[3.5rem] text-[#2c2e34] font-semibold">
```

Comment:
mobile font size for `value` should be same as `title` too!
feel free to merge when that's in

Reply:
Oops lol I missed out this one. Adjusted in 4533d9f168bbf9b97cef91e07788d24d50ee13d0.